pkgs/sway/default.nix

```
@@ -25,6 +25,9 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     # replace the version
+    sed -i "s/\([ \t]\)version: '\(.*\)',/\1version: '\2-${metadata.rev} (${metadata.revdate}, branch \\\'${metadata.branch}\\\')',/" \
```

No longer unnecessarily wordy (just need to bring `lib` or `substring` into scope) :)

```suggestion
    sed -i "s/\([ \t]\)version: '\(.*\)',/\1version: '\2-${lib.substring 0 8 metadata.rev} (${metadata.revdate}, branch \\\'${metadata.branch}\\\')',/" \
```

---

I'm still going to drop the date too, it's too easy to go from commit hash -> revdate for any user that wants to, either via git, github, or looking at metadata.nix.